react/react2/week1/search-app/git-app/src/components/SearchUser.js

```diff
@@ -0,0 +1,35 @@
+import React, { useEffect, useState } from 'react';
+import User from './User';
+
+function SearchUser({ username }) {
+  const [isLoading, setIsloading] = useState(true);
+  const [user, setuser] = useState([]);
+
+  useEffect(() => {
+    fetch(`https://api.github.com/search/users?q=${username}`)
+      .then((response) => response.json())
+      .then((data) => {
+        setIsloading(false);
```

Comment:
so you are setting loading to false, but you never set it to true. think, that loading should be true every time before data is received.
Also you are missing catch block, to catch errors, that you should display, if there is one